src/Console/Command/FixCommand.php

```diff
@@ -275,4 +288,23 @@ private function calculateExitStatus($isDryRun, $hasChangedFiles, $hasInvalidErr
 
         return $exitStatus;
     }
+
+    private function getColumns(ConfigurationResolver $resolver)
+    {
+        $columns = $resolver->getColumns();
+        if (null === $columns) {
+            return null;
+        }
+
+        if ('max' === $columns) {
+            // Terminal was introduced in symfony/console 3.2
+            if (!class_exists(Terminal::class)) {
+                return $this->getApplication()->getTerminalDimensions()[0];
+            }
+
+            return (new Terminal())->getWidth();
```

Comment:
`getTerminalDimensions()` has been deprecated and the notice suggests to use `Terminal` instead so it would be surprising that they don't work the same way.